cmd/ebrelayer/replay.go

```diff
@@ -216,3 +216,64 @@ func RunListMissedCosmosEventCmd(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
```

Comment:
Why is this code in replay.go?  Doesn't seem like it has anything to do with replay.

Reply:
accept, will rename the source file.